test/src/test/java/hudson/node_monitors/ResponseTimeMonitorTest.java

```diff
@@ -47,8 +48,21 @@ public void skipOfflineAgent() throws Exception {
         j.disconnectSlave(s);
         assertNull(ResponseTimeMonitor.DESCRIPTOR.monitor(c));
 
-        // Now reconnect and make sure we get a non-null response.
-        c.connect(false).get(); // wait until it's connected
+        // Retry to compensate for test being flaky in CI
+        int attempt = 0;
```

Comment:
much cleaner to use https://github.com/awaitility/awaitility/wiki/Usage#ignoring-exceptions ?

Reply:
Oh, right. I was not checking if `awaitility` is available. Give me a second...